src/lib/component/partial/EventsList/StatusMenu.js

```diff
@@ -15,23 +15,33 @@ import { ErrorHollowIcon } from "/lib/component/icon";
 const options = [
   {
     value: "incident",
-    Icon: () => <ErrorHollowIcon />,
+    Icon: function IncidentIcon() {
```

Comment:
Is this a TypeScript convention?

Reply:
This is to satisfy an eslint rule that asserts components have display names.